apps/sensenet/src/services/MonacoModels/PersonalSettingsModel.ts

```diff
@@ -247,8 +247,8 @@ export const setupModel = (language = defaultLanguage, repo: Repository) => {
                             settings: {
                               type: 'object',
                               properties: {
-                                term: { type: 'string', description: language.drawer.contentRootDescription },
-                                columns: { $ref: '#/definitions/columns' },
+                                term: { type: 'string', description: language.drawer.queryTerm },
+                                columns: { $ref: '#/definitions/columns', description: language.drawer.columns },
```

Comment:
This line can causes the duplicated description on hover.  ?!

Reply:
Indeed. Good catch! Thanks 😊